src/main/java/ca/corefacility/bioinformatics/irida/ria/web/search/SearchController.java

```diff
@@ -1,148 +1,107 @@
-	 * @return a {@link DataTablesResponse} to display the search results
-	 */
-	@RequestMapping("/search/ajax/projects")
-	@ResponseBody
-	public DataTablesResponse searchProjects(@RequestParam String query,
-			@RequestParam(required = false, defaultValue = "false") boolean global,
-			@DataTablesRequest DataTablesParams params) {
-		Page<Project> page;
-		if (global) {
-			page = projectService.findAllProjects(query, params.getCurrentPage(), params.getLength(), params.getSort());
-		} else {
-			page = projectService.findProjectsForUser(query, params.getCurrentPage(), params.getLength(),
-					params.getSort());
-		}
-		List<DataTablesResponseModel> projects = page.getContent().stream().map(this::createDataTablesProject)
-				.collect(Collectors.toList());
-		return new DataTablesResponse(params, page, projects);
-	}
-
-	/**
-	 * Search all {@link Sample}s in projects for a user based on a query string
-	 *
-	 * @param query  the query string
-	 * @param global Whether to perform an admin
-	 *               global search
-	 * @param params parameters for a datatables response
-	 * @return a {@link DataTablesResponse} to display search results
-	 */
-	@RequestMapping("/search/ajax/samples")
-	@ResponseBody
-	public DataTablesResponse searchSamples(@RequestParam String query,
-			@RequestParam(required = false, defaultValue = "false") boolean global,
-			@DataTablesRequest DataTablesParams params) {
+    private final ProjectService projectService;
+    private final SampleService sampleService;
 
-		Sort originalSort = params.getSort();
-		List<Sort.Order> orders = Lists.newArrayList();
-		originalSort.forEach(o -> {
-			orders.add(new Sort.Order(o.getDirection(), "sample." + o.getProperty()));
-		});
+    @Autowired
+    public SearchController(ProjectService projectService, SampleService sampleService) {
+        this.projectService = projectService;
+        this.sampleService = sampleService;
+    }
 
-		Sort newSort = Sort.by(orders);
-		Page<ProjectSampleJoin> samplePage;
-		if (global) {
-			samplePage = sampleService.searchAllSamples(query, params.getCurrentPage(), params.getLength(), newSort);
-		} else {
-			samplePage = sampleService.searchSamplesForUser(query, params.getCurrentPage(), params.getLength(),
-					newSort);
-		}
+    /**
+     * Get the search view
+     *
+     * @return name of the search view
+     */
+    @RequestMapping("/search")
+    public String search() {
+        return "search/search";
+    }
 
-		List<DataTablesResponseModel> samples = samplePage.getContent().stream().map(this::createDataTablesSample)
-				.collect(Collectors.toList());
-		return new DataTablesResponse(params, samplePage, samples);
-	}
+    /**
+     * Search all projects a user is a member of based on a query string
+     *
+     * @param request Details about the search request
+     * @return Paged list of projects and the total count found for the search
+     */
+    @PostMapping("/ajax/search/projects")
```

Comment:
Change this to either RequestMapping or GetMapping. We aren't posting anything to the server here

Reply:
Updated in cb492830eea345273c958248b3b34e4891c4ab38